test/decl/var/property_wrappers.swift

```diff
@@ -1091,3 +1091,18 @@ struct InvalidPropertyDelegateUse {
     self.x.foo() // expected-error {{value of type 'Int' has no member 'foo'}}
   }
 }
+
+// SR-11060
+
+class SR_11060_Class {
+  @SR_11060_Wrapper var property: Int = 1234 // expected-error {{missing argument for parameter 'string' in call}}{{20-20=(string: <#String#>)}}
```

Comment:
I think it would be fine for now just make it so diagnostic doesn't produce any fix-its, since fix-it has to mention all of the arguments, correct one would be to remove `= 123` and make add `(wrappedValue: 123, string: <#String#>)` after attribute.

Reply:
Hmm it would be confusing for someone new to figure out where the arguments go when they see this diagnostic without a fix-it. Once we add this fix-it, the existing logic in `missingArgument` takes care of any other missing arguments and insert them into the correct location.

Reply:
We can change the diagnostic text though to make it more obvious.

Reply:
Yeah, i think tailored diagnostic for this case would be nice. I'm not sure how hard it would be but if we could detect that `=` is used when implicit constructor has more parameters we could diagnose this case even earlier and not even get constraint system involved.

Reply:
Okay, how about something like this: `missing argument for parameter 'foo' in property wrapper initializer; add 'wrappedValue' and 'foo' arguments in @FooBar(...)`

Reply:
Yeah, that sounds good to me!

Reply:
@xedin Ok done :)

Reply:
> I think it would be fine for now just make it so diagnostic doesn't produce any fix-its, since fix-it has to mention all of the arguments, correct one would be to remove `= 123` and make add `(wrappedValue: 123, string: <#String#>)` after attribute.

@xedin I agree with your decision to get rid of the fix-it. I however had a couple of questions on the fix-it, if it were present:
- The fix-it for simple functions reports only one missing argument at a time, so why wouldn't it be okay here too?
- Wouldn't the correct fix-it be to keep the `= 123` and add `(string: <#String#>)` after the `@SR_11060_Wrapper`, which would be the recommended use of property wrappers?

Reply:
That's how I [did it](https://github.com/apple/swift/pull/26219/commits/a3b6ac555cdb20b5c7e7c686bc1de085a89ef52e) initially - the property wrappers [proposal](https://github.com/DougGregor/swift-evolution/blob/property-wrappers/proposals/0258-property-wrappers.md) has many examples where arguments are assigned inside '@Whatever(...)` and the initial value is assigned using `=`. I don't think I have seen an example where all of them are assigned inside '@Whatever(...)`.

Reply:
I just don't think it's practical to do so in CSDiag, once we get proper diagnostics for this kind of problem in new diagnostic framework we can provide a proper fix-it since it'd have information about all of the types involved.

Reply:
That would be great!